__init__.py

```diff
@@ -15,7 +15,7 @@
 def create_app():
     app = Flask(__name__)
     # app.config["SECRET_KEY"] = os.environ
-    app.secret_key = os.getenv("SECRET_KEY")
+    app.secret_key = os.getenv("SECRET_KEY") or os.urandom(24)
```

Comment:
we should avoid keeping the secret key in the code, otherwise the csrf security is worthless